cmd/fs-v1.go

```diff
@@ -42,6 +42,9 @@ type fsObjects struct {
 	// temporary transactions.
 	fsUUID string
 
+	// This value shouldn't be touched, once initialized.
+	fsFormatRlk *lock.RLockedFile // Is a read lock remembered on `format.json`.
```

Comment:
suggestion: "Is a read lock on `format.json`" reads better. The value being remembered is implicit.